proto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "shuttle-proto"
-version.workspace = true
+version = "0.42.0"
```

Comment:
Is this expected?

Reply:
I made it follow the same pattern as the rest of the crates. No real world difference

Reply:
Oky